diff -urN ircii-4.4L.orig/Makefile.in ircii-4.4L/Makefile.in
--- ircii-4.4L.orig/Makefile.in	Fri Sep  3 18:25:07 1999
+++ ircii-4.4L/Makefile.in	Fri Sep  3 18:29:20 1999
@@ -59,17 +59,18 @@
 #   clean               - remove all .o files, core, and binaries
 #   distclean           - remove all files geneated by compilation/installation.
 
-prefix = @prefix@
-exec_prefix = @exec_prefix@
+DESTDIR=
+prefix = $(DESTDIR)@prefix@
+exec_prefix = $(DESTDIR)@exec_prefix@
 topdir = @srcdir@
 srcdir = @srcdir@/source
 incdir = @srcdir@/include
 VPATH = @srcdir@/source
 objdir = @objdir@
-bindir = @bindir@
-datadir = @datadir@
+bindir = $(DESTDIR)@bindir@
+datadir = $(DESTDIR)@datadir@
 libdir = ${exec_prefix}/lib
-mandir = ${prefix}/man/man1
+mandir = ${prefix}/share/man/man1
 
 # Where the ircII binary will be installed.
 # 'make install' will compile and install the program
@@ -153,16 +154,16 @@
 
 # Set the next line to the full path for installation of the ircio program
 # if you wish to use it. 
-INSTALL_IRCIO = $(bindir)/ircio
+INSTALL_IRCIO = $(bindir)/ircio-ircii
 
 # This little program is necessary to have an interactive shell
 # in a window of ircII. The 'shell' script uses it, so also update
 # the path in there, if you want to enable this.
-INSTALL_IRCFLUSH = $(bindir)/ircflush
+INSTALL_IRCFLUSH = $(bindir)/ircflush-ircii
 
 # This program allows you to use screen/xterm's to put new irc windows
 # on new screen/xterm windows.
-INSTALL_WSERV = $(bindir)/wserv
+INSTALL_WSERV = $(bindir)/wserv-ircii
 
 # This command will be used to install the irc help files.  If you don't
 # want to install them, replace with the following:
@@ -196,7 +197,7 @@
 
 installirc: irc installdirs
 	$(INSTALL) irc $(INSTALL_IRC)-$(VERSION)
-	$(INSTALL) ircbug $(bindir)/ircbug
+	$(INSTALL) ircbug $(bindir)/ircbug-ircii
 	@if test -f $(INSTALL_IRC).old; then $(RM) $(INSTALL_IRC).old; fi
 	@if test -f $(INSTALL_IRC); then $(MV) $(INSTALL_IRC) $(INSTALL_IRC).old; fi
 	$(RM) $(INSTALL_IRC)
diff -urN ircii-4.4L.orig/configure ircii-4.4L/configure
--- ircii-4.4L.orig/configure	Fri Sep  3 18:25:07 1999
+++ ircii-4.4L/configure	Fri Sep  3 18:26:27 1999
@@ -3177,7 +3177,7 @@
   cat > conftest.$ac_ext <<EOF
 #line 3179 "configure"
 #include "confdefs.h"
-#include <curses.h>
+#include <ncurses/curses.h>
 
 EOF
 if (eval "$ac_cpp conftest.$ac_ext") 2>&5 |
diff -urN ircii-4.4L.orig/configure.in ircii-4.4L/configure.in
--- ircii-4.4L.orig/configure.in	Fri Sep  3 18:25:07 1999
+++ ircii-4.4L/configure.in	Fri Sep  3 18:26:51 1999
@@ -375,7 +375,7 @@
         AC_CHECK_LIB(xtermcap, tgetent, LIBS="-lxtermcap $olibs",
           AC_MSG_WARN(i can not find tgetent.  you might need to help me)))))
   AC_MSG_CHECKING(for tputs in curses.h)
-  AC_EGREP_CPP([tputs( |	|\()],[#include <curses.h>
+  AC_EGREP_CPP([tputs( |	|\()],[#include <ncurses/curses.h>
 ], AC_DEFINE(INCLUDE_CURSES_H) AC_MSG_RESULT(yes), blahhack=1; AC_MSG_RESULT(no))
 fi
 
@@ -699,7 +699,7 @@
 
 if test ! -n "$IRCLIB";
 then
-	IRCLIB=\${datadir}/irc
+	IRCLIB=\${datadir}/ircii
 fi
 AC_SUBST(IRCLIB)
 
diff -urN ircii-4.4L.orig/include/config.h.dist ircii-4.4L/include/config.h.dist
--- ircii-4.4L.orig/include/config.h.dist	Fri Sep  3 18:25:09 1999
+++ ircii-4.4L/include/config.h.dist	Fri Sep  3 18:32:46 1999
@@ -47,7 +47,7 @@
  * file and the ircII can find it, this setting is overridden.
  */
 #ifndef DEFAULT_SERVER
-# define DEFAULT_SERVER	    "change.this.to.a.server"
+# define DEFAULT_SERVER	    "warszawa.irc.pl"
 #endif
 
 /*
@@ -89,7 +89,7 @@
  * DEFAULT_SERVER. 
 */
 
-#define SERVERS_FILE "ircII.servers"
+#define SERVERS_FILE "/etc/irc/ircII.servers"
 
 /* Uncomment the following if you want ircII to display the file
  * ircII.motd in the ircII library at startup.
@@ -131,7 +131,7 @@
 #define DEFAULT_CTCP_REPLY_IGNORE_SECONDS 10
 #define DEFAULT_DCC_BLOCK_SIZE 512
 #define DEFAULT_DISPLAY 1
-#define DEFAULT_EIGHT_BIT_CHARACTERS 0
+#define DEFAULT_EIGHT_BIT_CHARACTERS 1
 #define DEFAULT_ENCRYPT_PROGRAM NULL
 #define DEFAULT_EXEC_PROTECTION 1
 #define DEFAULT_FLOOD_AFTER 3
@@ -157,7 +157,7 @@
 #define DEFAULT_LASTLOG 44
 #define DEFAULT_LASTLOG_LEVEL "ALL -CRAP"
 #define DEFAULT_LOG 0
-#define DEFAULT_LOGFILE "IrcLog"
+#define DEFAULT_LOGFILE ".IrcLog"
 #define DEFAULT_MAIL 0
 #define DEFAULT_MAX_RECURSIONS 10
 #define DEFAULT_MINIMUM_SERVERS 0
diff -urN ircii-4.4L.orig/include/ircterm.h ircii-4.4L/include/ircterm.h
--- ircii-4.4L.orig/include/ircterm.h	Fri Sep  3 18:25:09 1999
+++ ircii-4.4L/include/ircterm.h	Fri Sep  3 18:26:27 1999
@@ -38,7 +38,7 @@
 # define __ircterm_h_
 
 #ifdef INCLUDE_CURSES_H
-# include <curses.h>
+# include <ncurses/curses.h>
 #endif /* INCLUDE_CURSES_H */
 
 /* this is really busted on solaris -mrg */
