diff -urNp -x '*.orig' ircii-20111115.org/Makefile.in ircii-20111115/Makefile.in
--- ircii-20111115.org/Makefile.in	2011-07-10 08:18:01.000000000 +0200
+++ ircii-20111115/Makefile.in	2023-02-02 08:55:48.090150078 +0100
@@ -109,16 +109,16 @@ TRANS_PATH = $(IRCLIB)/translation
 
 # Set the next line to the full path for installation of the ircio program
 # if you wish to use it. 
-INSTALL_IRCIO = $(libexecdir)/ircio
+INSTALL_IRCIO = $(bindir)/ircio-ircii
 
 # This little program is necessary to have an interactive shell
 # in a window of ircII. The 'shell' script uses it, so also update
 # the path in there, if you want to enable this.
-INSTALL_IRCFLUSH = $(bindir)/ircflush
+INSTALL_IRCFLUSH = $(bindir)/ircflush-ircii
 
 # This program allows you to use screen/xterm's to put new irc windows
 # on new screen/xterm windows.
-INSTALL_WSERV = $(libexecdir)/wserv
+INSTALL_WSERV = $(bindir)/wserv-ircii
 
 # This command will be used to install the irc help files.  If you don't
 # want to install them, replace with the following:
@@ -154,7 +154,7 @@ installbin: installirc installio install
 
 installirc: irc installdirs
 	$(INSTALL) irc $(INSTALL_IRC)-$(VERSION)
-	$(INSTALL) ircbug $(bindir)/ircbug
+	$(INSTALL) ircbug $(bindir)/ircbug-ircii
 	@if test -f $(INSTALL_IRC).old; then $(RM) $(INSTALL_IRC).old; fi
 	@if test -f $(INSTALL_IRC); then $(MV) $(INSTALL_IRC) $(INSTALL_IRC).old; fi
 	$(RM) $(INSTALL_IRC)
diff -urNp -x '*.orig' ircii-20111115.org/configure.in ircii-20111115/configure.in
--- ircii-20111115.org/configure.in	2011-11-14 22:57:23.000000000 +0100
+++ ircii-20111115/configure.in	2023-02-02 08:55:48.090150078 +0100
@@ -483,7 +483,7 @@ AC_SUBST(bindir)
 
 if test ! -n "$IRCLIB";
 then
-	IRCLIB=\${datadir}/irc
+	IRCLIB=\${datadir}/ircii
 fi
 AC_SUBST(IRCLIB)
 
diff -urNp -x '*.orig' ircii-20111115.org/include/config.h.dist ircii-20111115/include/config.h.dist
--- ircii-20111115.org/include/config.h.dist	2008-10-28 17:22:48.000000000 +0100
+++ ircii-20111115/include/config.h.dist	2023-02-02 08:55:48.090150078 +0100
@@ -47,7 +47,7 @@
  * file and the ircII can find it, this setting is overridden.
  */
 #ifndef DEFAULT_SERVER
-# define DEFAULT_SERVER	    "change.this.to.a.server"
+# define DEFAULT_SERVER	    "poznan.irc.pl"
 #endif
 
 /*
@@ -89,7 +89,7 @@
  * DEFAULT_SERVER. 
 */
 
-#define SERVERS_FILE "ircII.servers"
+#define SERVERS_FILE "/etc/irc/ircII.servers"
 
 /* Uncomment the following if you want ircII to display the file
  * ircII.motd in the ircII library at startup.
@@ -164,7 +164,7 @@
 #define DEFAULT_LASTLOG 440
 #define DEFAULT_LASTLOG_LEVEL "ALL -CRAP"
 #define DEFAULT_LOG 0
-#define DEFAULT_LOGFILE "IrcLog"
+#define DEFAULT_LOGFILE ".irciilog"
 #define DEFAULT_MAIL 0
 #define DEFAULT_MAKE_NOTICE_MSG 1
 #define DEFAULT_MAX_RECURSIONS 10
