diff -urN ircii-4.4J.orig/Makefile.in ircii-4.4J/Makefile.in
--- ircii-4.4J.orig/Makefile.in	Sun Aug  1 12:18:08 1999
+++ ircii-4.4J/Makefile.in	Sun Aug  1 12:38:32 1999
@@ -49,11 +49,12 @@
 #   clean               - remove all .o files, core, and binaries
 #   distclean           - remove all files geneated by compilation/installation.
 
-prefix = @prefix@
-exec_prefix = @exec_prefix@
-bindir = @bindir@
+DESTDIR=
+prefix = ${DESTDIR}@prefix@
+exec_prefix = ${DESTDIR}@exec_prefix@
+bindir = ${DESTDIR}@bindir@
 libdir = ${exec_prefix}/lib
-mandir = ${prefix}/man/man1
+mandir = ${prefix}/share/man/man1
 
 # Where the ircII binary will be installed.
 # 'make install' will compile and install the program
@@ -137,16 +138,16 @@
 
 # Set the next line to the full path for installation of the ircio program
 # if you wish to use it. 
-INSTALL_IRCIO = $(bindir)/ircio
+INSTALL_IRCIO = $(bindir)/ircio-ircii
 
 # This little program is necessary to have an interactive shell
 # in a window of ircII. The 'shell' script uses it, so also update
 # the path in there, if you want to enable this.
-INSTALL_IRCFLUSH = $(bindir)/ircflush
+INSTALL_IRCFLUSH = $(bindir)/ircflush-ircii
 
 # This program allows you to use screen/xterm's to put new irc windows
 # on new screen/xterm windows.
-INSTALL_WSERV = $(bindir)/wserv
+INSTALL_WSERV = $(bindir)/wserv-ircii
 
 # This command will be used to install the irc help files.  If you don't
 # want to install them, replace with the following:
@@ -215,7 +216,7 @@
 
 installirc: irc installdirs
 	$(INSTALL) source/irc $(INSTALL_IRC)-$(VERSION)
-	$(INSTALL) ircbug $(bindir)/ircbug
+	$(INSTALL) ircbug $(bindir)/ircbug-ircii
 	@if test -f $(INSTALL_IRC).old; then $(RM) $(INSTALL_IRC).old; fi
 	@if test -f $(INSTALL_IRC); then $(MV) $(INSTALL_IRC) $(INSTALL_IRC).old; fi
 	$(RM) $(INSTALL_IRC)
diff -urN ircii-4.4J.orig/configure ircii-4.4J/configure
--- ircii-4.4J.orig/configure	Sun Aug  1 12:18:08 1999
+++ ircii-4.4J/configure	Sun Aug  1 12:43:57 1999
@@ -3121,7 +3121,7 @@
   cat > conftest.$ac_ext <<EOF
 #line 3123 "configure"
 #include "confdefs.h"
-#include <curses.h>
+#include <ncurses/curses.h>
 
 EOF
 if (eval "$ac_cpp conftest.$ac_ext") 2>&5 |
@@ -3966,7 +3966,7 @@
 
 if test ! -n "$IRCLIB";
 then
-	IRCLIB=\${prefix}/share/irc
+	IRCLIB=\${prefix}/share/ircii
 fi
 
 if test -n "$nextstep";
diff -urN ircii-4.4J.orig/configure.in ircii-4.4J/configure.in
--- ircii-4.4J.orig/configure.in	Sun Aug  1 12:18:08 1999
+++ ircii-4.4J/configure.in	Sun Aug  1 12:43:41 1999
@@ -351,7 +351,7 @@
         AC_CHECK_LIB(xtermcap, tgetent, LIBS="-lxtermcap $olibs",
           AC_MSG_WARN(i can not find tgetent.  you might need to help me)))))
   AC_MSG_CHECKING(for tputs in curses.h)
-  AC_EGREP_CPP([tputs( |	|\()],[#include <curses.h>
+  AC_EGREP_CPP([tputs( |	|\()],[#include <ncurses/curses.h>
 ], AC_DEFINE(INCLUDE_CURSES_H) AC_MSG_RESULT(yes), blahhack=1; AC_MSG_RESULT(no))
 fi
 
@@ -674,7 +674,7 @@
 
 if test ! -n "$IRCLIB";
 then
-	IRCLIB=\${prefix}/share/irc
+	IRCLIB=\${prefix}/share/ircii
 fi
 
 if test -n "$nextstep";
diff -urN ircii-4.4J.orig/include/config.h ircii-4.4J/include/config.h
--- ircii-4.4J.orig/include/config.h	Sun Aug  1 12:18:09 1999
+++ ircii-4.4J/include/config.h	Sun Aug  1 12:21:01 1999
@@ -47,7 +47,7 @@
  * file and the ircII can find it, this setting is overridden.
  */
 #ifndef DEFAULT_SERVER
-# define DEFAULT_SERVER	    "change.this.to.a.server"
+# define DEFAULT_SERVER	    "warszawa.irc.pl"
 #endif
 
 /*
@@ -77,7 +77,7 @@
  * If you undefine this you will spare some code, though, so better only
  * set if you are sure you are going to keep your help-files compressed.
  */
-#define ZCAT "/usr/ucb/zcat"
+#define ZCAT "/bin/zcat"
 
 /* Define ZSUFFIX in case we are using ZCAT */
 #ifdef ZCAT
@@ -101,7 +101,7 @@
  * DEFAULT_SERVER. 
 */
 
-#define SERVERS_FILE "ircII.servers"
+#define SERVERS_FILE "/etc/irc/ircII.servers"
 
 /* Uncomment the following if you want ircII to display the file
  * ircII.motd in the ircII library at startup.
@@ -143,7 +143,7 @@
 #define DEFAULT_CTCP_REPLY_IGNORE_SECONDS 10
 #define DEFAULT_DCC_BLOCK_SIZE 512
 #define DEFAULT_DISPLAY 1
-#define DEFAULT_EIGHT_BIT_CHARACTERS 0
+#define DEFAULT_EIGHT_BIT_CHARACTERS 1
 #define DEFAULT_ENCRYPT_PROGRAM NULL
 #define DEFAULT_EXEC_PROTECTION 1
 #define DEFAULT_FLOOD_AFTER 3
@@ -169,7 +169,7 @@
 #define DEFAULT_LASTLOG 44
 #define DEFAULT_LASTLOG_LEVEL "ALL -CRAP"
 #define DEFAULT_LOG 0
-#define DEFAULT_LOGFILE "IrcLog"
+#define DEFAULT_LOGFILE ".IrcLog"
 #define DEFAULT_MAIL 0
 #define DEFAULT_MAX_RECURSIONS 10
 #define DEFAULT_MINIMUM_SERVERS 0
diff -urN ircii-4.4J.orig/include/ircterm.h ircii-4.4J/include/ircterm.h
--- ircii-4.4J.orig/include/ircterm.h	Sun Aug  1 12:18:09 1999
+++ ircii-4.4J/include/ircterm.h	Sun Aug  1 12:44:29 1999
@@ -38,7 +38,7 @@
 # define __ircterm_h_
 
 #ifdef INCLUDE_CURSES_H
-# include <curses.h>
+# include <ncurses/curses.h>
 #endif /* INCLUDE_CURSES_H */
 
 /* this is really busted on solaris -mrg */
